### PR TITLE
Crm 23 customers table

### PR DESCRIPTION
**Pull request by magikabdul**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import GlobalStyles from './theme/GlobalStyles';
+import { ThemeProvider } from 'styled-components';
```
Comment (Fellway) [inline frontend/src/App.js:3]: image

File: frontend/src/components/atoms/TableCell/TableCell.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  
+  text-align: left;
+`;
+
+const TableCell = ({ children }) => (
+  <Cell>{children}</Cell>
+);
+
+TableCell.propTypes = {
+  children: PropTypes.node.isRequired,
```
Comment (Fellway) [inline frontend/src/components/atoms/TableCell/TableCell.js:21]: Why node?

File: frontend/src/components/atoms/TableCell/TableCell.stories.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import TableCell from './TableCell';
+
+const DivContainer = styled.div`
+  display: flex;
+  justify-content: center;
+
```
Comment (Fellway) [inline frontend/src/components/atoms/TableCell/TableCell.stories.js:8]: remove empty spaces

File: frontend/src/components/atoms/TableHeaderCell/TableHeaderCell.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+const Container = styled.div`
+  margin: 0 8px;
+  padding: 8px;
+  
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  
+  color: ${({ color, theme }) => {
```
Comment (Fellway) [inline frontend/src/components/atoms/TableHeaderCell/TableHeaderCell.js:13]: Move this function to utils module. This function can be used in different places.

File: frontend/src/components/organisms/Table/Table.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+  background-color: ${({ theme }) => theme.color.grey};
+`;
+
+const TableContainer = styled.table`
+  width: 100%;
+  border-spacing: 0;
+`;
+
+const TableBody = styled.tbody`
```
Comment (Fellway) [inline frontend/src/components/organisms/Table/Table.js:20]: What is it? Remove if its empty.

File: frontend/src/components/organisms/Table/Table.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Header from '../../atoms/Header/Header';
+import TableRow from '../../molecules/TableRow/TableRow';
+import TableHeader from '../../molecules/TableHeader/TableHeader';
+import TableCell from '../../atoms/TableCell/TableCell';
+
+const Wrapper = styled.div`
```
Comment (Fellway) [inline frontend/src/components/organisms/Table/Table.js:9]: Whole Table should use shadow.
Please use: 
image

Comment (Fellway) [inline frontend/src/components/organisms/Table/Table.js:9]: Remove default table style. Table should be without border and with given shadows.
image

**Review by Fellway — CHANGES_REQUESTED** [review]

File: frontend/src/App.js (hunk 1 above)
Comment (magikabdul) [reply frontend/src/App.js:3]: fixed

File: frontend/src/components/atoms/TableCell/TableCell.stories.js (hunk 3 above)
Comment (magikabdul) [reply frontend/src/components/atoms/TableCell/TableCell.stories.js:8]: removed

File: frontend/src/components/atoms/TableHeaderCell/TableHeaderCell.js (hunk 4 above)
Comment (magikabdul) [reply frontend/src/components/atoms/TableHeaderCell/TableHeaderCell.js:13]: moved

File: frontend/src/components/organisms/Table/Table.js (hunk 5 above)
Comment (magikabdul) [reply frontend/src/components/organisms/Table/Table.js:20]: removed

File: frontend/src/components/organisms/Table/Table.js (hunk 6 above)
Comment (magikabdul) [reply frontend/src/components/organisms/Table/Table.js:9]: removed

Comment (magikabdul) [reply frontend/src/components/organisms/Table/Table.js:9]: done

File: frontend/src/components/atoms/TableCell/TableCell.js (hunk 2 above)
Comment (magikabdul) [reply frontend/src/components/atoms/TableCell/TableCell.js:21]: changed to element

**Review by Fellway — APPROVED** [review]

**Pull request merged**